src/core_plugins/kibana/server/tutorials/apm/apm_server_instructions.js

```diff
@@ -122,8 +122,8 @@ PowerShell.\n5. From the PowerShell prompt, run the following commands to instal
         }
       }),
       commands: [
-        `PS > cd 'C:\\Program Files\\APM-Server'`,
-        `PS C:\\Program Files\\APM-Server> .\\install-service-apm-server.ps1`,
+        `cd 'C:\\Program Files\\APM-Server'`,
```

Comment:
Should we change these to double quotes to stay in keeping with the other versions above?
`cd "C:\\Program Files\\APM-Server"`?

Reply:
I'm not terribly worried about consistency there so long as it works.